### ISSUE TITLE
Card Expiry month / year is of type long?

### ISSUE THREAD

**Issue by clement911** [stripe/stripe-dotnet#1929]
Card Expiry month / year is of type long?

Why not int? Or even byte for month, short for year?

**Comment by remi-stripe** [comment]
@clement911 In the library, we use `long` consistently for all integers. It's just easier if all numbers are either `decimal` or `long` and you don't have to worry about the type. We could for sure optimize the size and use `short` but in an application using stripe-dotnet, this shouldn't change much. Consistency makes it easier for developers to understand how to use our APIs.

**Comment by clement911** [comment]
Personally I don't really see how it makes it easier to have `long`s everywhere.
On the contrary, it confused me, hence why I had to ask.
It's not for optimization purposes that I was asking but rather for self documentation (data type is a form a code as documentation if you will). I had to wonder why would I ever use a year bigger than `int.MaxValue`

(One thing somewhat related is that `long.MaxValue` cannot be represented in JSON as a number so some serializer will serialize longs as a string)
